schedule/qam/common/virt/qam_virt_install_guest.yaml

```diff
@@ -6,5 +6,24 @@ schedule:
   - virt_autotest/login_console
   - virtualization/universal/prepare_guests
   - virtualization/universal/waitfor_guests
+  - '{{patch_host_zypper}}'
```

Comment:
If the MU is kernel It will patch both host and guest. If the MU is xen/libvirt/qemu it will patch host only, not patch guest. 
Do you have better name?

Reply:
what I can think of is adding guest in the name as well.

Reply:
"patch_h_and_g_zypper" "patch_host_guest_zypper"? which is better?

Reply:
Personally I'd like the latter. It is a minor issue, depends on you.

Reply:
changed